infra/blueprint-test/test/terraform_simple_base_api_module_test.go

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/modules/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/modules/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/modules/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSimpleTFModule(t *testing.T) {
+	nt := tft.Init(t,
+		tft.WithTFDir("../examples/simple_tf_module"),
+		tft.WithSetupPath("setup/simple_tf_module"),
+		tft.WithEnvVars(map[string]string{"network_name": fmt.Sprintf("foo-%s", utils.RandStr(5))}),
+	)
+
+	utils.RunStage("setup", func() { nt.Setup() })
+	defer utils.RunStage("teardown", func() { nt.Teardown() })
+
+	utils.RunStage("apply", func() { nt.Apply() })
+
+	utils.RunStage("verify", func() {
```

Comment:
Should this nt.Verify() first to establish that as the common convention?

Reply:
added